streams/src/test/java/org/apache/kafka/streams/kstream/internals/InternalStreamsBuilderTest.java

```diff
@@ -156,6 +156,30 @@ public void shouldStillMaterializeSourceKTableIfMaterializedIsntQueryable() thro
         assertEquals("topic2", topology.storeToChangelogTopic().get(storeName));
         assertNull(table1.queryableStoreName());
     }
+    
+    @Test
+    public void shouldBuildGlobalTableWithNonQueryableStoreName() throws Exception {
+        final GlobalKTable<String, String> table1 = builder.globalTable("topic2",
```

Comment:
I think, `builder.globalTable("anyTopic")` should be sufficient here?

Reply:
There doesn't exist any ```globalTable``` method with only a ```string``` as an input argument. The client must also provide a ```Materialized``` at the vert least.

Reply:
https://docs.confluent.io/current/streams/javadocs/org/apache/kafka/streams/StreamsBuilder.html#globalTable(java.lang.String)

Reply:
Ah. This is testing `InternalStreamsBuilder` -- by bad -- I though this is testing `StreamsBuilder`.